attributes/default.rb

```diff
@@ -179,7 +179,7 @@
 
 # Configure Abiquo websockify
 default['abiquo']['websockify']['port'] = 41338
-default['abiquo']['websockify']['address'] = '127.0.0.1'
+default['abiquo']['websockify']['address'] = '0.0.0.0'
 default['abiquo']['websockify']['api_url'] = 'https://localhost/api'
```

Comment:
If you mean this attribute, I would say this is for the old script and should be already removed. Right now the noVNC plugin gets the full VM URL.